charts/camunda-platform/test/identity/golden/deployment.golden.yaml

```diff
@@ -32,19 +32,37 @@ spec:
         app.kubernetes.io/instance: camunda-platform-test
         app.kubernetes.io/managed-by: Helm
         app.kubernetes.io/part-of: camunda-platform
-        app.kubernetes.io/version: "SNAPSHOT"
+        app.kubernetes.io/version: "8.0.0-rc1"
         app.kubernetes.io/component: identity
     spec:
       containers:
       - name: identity
-        image: "camunda/identity:SNAPSHOT"
+        image: "camunda/identity:8.0.0-rc1"
         env:
           - name: KEYCLOAK_USERS_0_USERNAME
             value: "demo"
           - name: KEYCLOAK_USERS_0_PASSWORD
             value: "demo"
           - name: KEYCLOAK_USERS_0_ROLES_0
             value: "Identity"
+          - name: KEYCLOAK_USERS_0_ROLES_1
+            value: "Operate"
+          - name: KEYCLOAK_INIT_OPERATE_SECRET
+            valueFrom:
+              secretKeyRef:
+                name: "camunda-platform-test-operate-identity-secret"
+                key: operate-secret
+          - name: KEYCLOAK_INIT_OPERATE_ROOT_URL
+            value: "http://localhost:8080"
+          - name: KEYCLOAK_USERS_0_ROLES_2
+            value: "Tasklist"
+          - name: KEYCLOAK_INIT_TASKLIST_SECRET
+            valueFrom:
+              secretKeyRef:
+                name: "camunda-platform-test-tasklist-identity-secret"
+                key: tasklist-secret
+          - name: KEYCLOAK_INIT_TASKLIST_ROOT_URL
+            value: "http://localhost:8080"
```

Comment:
The port `8080` is used for all three services. Identity uses this configuration to set the frontend url (f.e. login callback url for OAuth), so if there will be another routing to another port/domains, then `IDENTITY_URL` has to be set

Reply:
Sorry not 100% sure whether I can follow. Do you think it might be better if we just configure for all different ports and mention this in the notes to port-forward also to different ports? Then we adjust here the config as well?

Like:

8080 always for identity
8081 for operate
8082 for tasklist

etc.

Reply:
thanks for your input @dlavrenuek ! I reworked it a bit see here https://github.com/camunda/camunda-platform-helm/pull/271#issuecomment-1096514410